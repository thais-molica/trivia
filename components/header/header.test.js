import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Header from './index';

it('should render Header', () => {
  const { getByText } = render(<Header />);
  expect(getByText('Teste Dev Frontend')).toBeInTheDocument();
});