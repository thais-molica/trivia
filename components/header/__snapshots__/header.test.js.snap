// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<header
  className="jsx-9438985"
>
  <a
    className="jsx-9438985"
    href="/"
  >
    Teste Dev Frontend
  </a>
</header>
`;
